scripts/fix_headers.py

```diff
@@ -1,23 +1,47 @@
 #!/usr/bin/env python
 
 import os
+import re
 import sys
 from os.path import join, split, normpath, exists
 import subprocess
 import argparse
+from collections import defaultdict
+
+parser = argparse.ArgumentParser(description=
+"""
+Script to fix the headers in the repository source files.  Useful when
```

Comment:
Thanks for adding a description. It would be great if you could elaborate a little on what you mean by "fix". It finds the correct path and updates it?

Reply:
Yes, when you move headers or source files around, the include statements are wrong.  This script fixes those include statements.